priv/static/assets/app.swiftui.styles

```diff
@@ -1,216 +1 @@
-       line: 1,
-       module: "",
-       source: "listRowInsets(EdgeInsets())"
-     ],
-     [
-       {:EdgeInsets,
-        [
-          file: "core_components.swiftui.ex",
-          line: 1,
-          module: "",
-          source: "listRowInsets(EdgeInsets())"
-        ], []}
-     ]}
-  ],
-  "navigationSubtitle(:subtitle)" => [
-    {:navigationSubtitle,
-     [
-       file: "core_components.swiftui.ex",
-       line: 1,
-       module: "",
-       source: "navigationSubtitle(:subtitle)"
-     ], [:subtitle]}
-  ],
-  "navigationTitle(:title)" => [
-    {:navigationTitle,
-     [
-       file: "core_components.swiftui.ex",
-       line: 1,
-       module: "",
-       source: "navigationTitle(:title)"
-     ], [:title]}
-  ],
-  "sheet(isPresented: attr(\"presented\"), content: :content)" => [
-    {:sheet,
-     [
-       file: "core_components.swiftui.ex",
-       line: 1,
-       module: "",
-       source: "sheet(isPresented: attr(\"presented\"), content: :content)"
-     ],
-     [
-       [
-         isPresented: {:__attr__,
-          [
-            file: "core_components.swiftui.ex",
-            line: 1,
-            module: "",
-            source: "sheet(isPresented: attr(\"presented\"), content: :content)"
-          ], "presented"},
-         content: :content
-       ]
-     ]}
-  ],
-  "toolbar(content: :toolbar)" => [
-    {:toolbar,
-     [
-       file: "core_components.swiftui.ex",
-       line: 1,
-       module: "",
-       source: "toolbar(content: :toolbar)"
-     ], [[content: :toolbar]]}
-  ]
-}
\ No newline at end of file
+%{"alert(attr(\"title\"), isPresented: attr(\"presented\"), actions: :actions, message: :message)" => [{:alert, [], [{:__attr__, [], "title"}, [isPresented: {:__attr__, [], "presented"}, actions: :actions, message: :message]]}], "background(.blue)" => [{:background, [], [{:., [], [nil, :blue]}]}], "bold(true)" => [{:bold, [], [true]}], "buttonStyle(.borderedProminent)" => [{:buttonStyle, [], [{:., [], [nil, :borderedProminent]}]}], "controlSize(.large)" => [{:controlSize, [], [{:., [], [nil, :large]}]}], "font(.caption)" => [{:font, [], [{:., [], [nil, :caption]}]}], "foregroundStyle(.red)" => [{:foregroundStyle, [], [{:., [], [nil, :red]}]}], "foregroundStyle(Color(\"MyColor\"))" => [{:foregroundStyle, [], [{:Color, [], ["MyColor"]}]}], "frame(maxWidth: .infinity)" => [{:frame, [], [[maxWidth: {:., [], [nil, :infinity]}]]}], "hidden()" => [{:hidden, [], []}], "listRowBackground(:empty)" => [{:listRowBackground, [], [:empty]}], "listRowInsets(EdgeInsets())" => [{:listRowInsets, [], [{:EdgeInsets, [], []}]}], "navigationSubtitle(:subtitle)" => [{:navigationSubtitle, [], [:subtitle]}], "navigationTitle(:title)" => [{:navigationTitle, [], [:title]}], "sheet(isPresented: attr(\"presented\"), content: :content)" => [{:sheet, [], [[isPresented: {:__attr__, [], "presented"}, content: :content]]}], "toolbar(content: :toolbar)" => [{:toolbar, [], [[content: :toolbar]]}]}
```

Comment:
I think switching to the non-umbrella app may have resulted in changing the stylesheet config to not include metadata.

@bcardarella are we OK with that, or would we like to switch it back?

Reply:
the meta data only is necessary for logging but if you want to turn it back on:

```elixir
config :live_view_native_styleshset,
  annotations: true
```

Reply:
We only had this config locally, not inside of the Livebook. I'm personally fine to keep it off since it wouldn't have been seen by users originally.

Reply:
the only issue is if they were to inspect the stylesheet from the browser, it is in the compact format